css/main.css

```diff
@@ -0,0 +1,193 @@
+/* body */
+
+body {
+    background-color: #eceef1;
+    float: center;
```

Comment:
`float` can only have `left` or `right` as its value. `center` is not a valid `float` value.